### PR TITLE
Move windows utilities 

### PR DESCRIPTION
**Pull request by mrsmith0x00**
Move tools, which doesn't working in GNU/Linux, in `/opt`.

### CODE REVIEW

**Comment by mrsmith0x00** [comment]
@FFY00 I see, what you with @osospeed store WinHex in `/opt/windows/`. I want to separate working utilities from just archival (which doesn't work in GNU/Linux). In what directory it is better to store it?

**Comment by FFY00** [comment]
Sounds good. Let me know when this PR is ready for merge.

**Comment by mrsmith0x00** [comment]
I finished work, can merge.

**Comment by FFY00** [comment]
Please fix the errors:

```
packages/3proxy-win32/PKGBUILD:4:1: W062 Unsafe quotes
packages/3proxy-win32/PKGBUILD:5:1: W062 Unsafe quotes
packages/3proxy-win32/PKGBUILD:23:1: E060 No final newline
```

File: packages/3proxy-win32/PKGBUILD [hunk 1]
```diff
@@ -20,4 +20,5 @@ package() {
   rm 3proxy-${pkgver}.zip
 
   cp -a * "$pkgdir/opt/windows/archive/3proxy-win32"
+
```
Comment (FFY00) [inline packages/3proxy-win32/PKGBUILD]: The line isn't here, it's in the end of the script.

Comment (mrsmith0x00) [reply packages/3proxy-win32/PKGBUILD]: Sorry, fatigue affects :)

**Pull request merged**